census/census.ipynb

```diff
@@ -611,8 +616,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "population = dynspread(datashade(points, cmap=fire, element_type=gv.Image))\n",
-    "gts.EsriImagery() * population * gts.StamenLabels().options(level=\"annotation\")"
+    "population = dynspread(rasterize(points, cmap=fire, aggregator='count')).opts(cnorm='eq_hist')\n",
```

Comment:
```suggestion
    "population = dynspread(rasterize(points, cmap=fire, aggregator='count')).opts(cnorm='eq_hist', tools=['hover'])\n",
```
(Now that it uses rasterize, should it also enable hover so that we can see the actual population counts per pixel?)

Reply:
Yep, my mistake.